google/cloud/spanner_dbapi/connection.py

```diff
@@ -37,11 +34,46 @@ class Connection:
     """
 
     def __init__(self, instance, database):
+        self._pool = BurstyPool()
+        self._pool.bind(database)
+
         self.instance = instance
         self.database = database
-        self.is_closed = False
 
         self._ddl_statements = []
+        self.transactions = []
+
+        self.is_closed = False
+        self._autocommit = False
+
+    @property
+    def autocommit(self):
+        """Autocommit mode flag for this connection.
+
+        :rtype: bool
+        :returns: Autocommit mode flag value.
+        """
+        return self._autocommit
+
+    @autocommit.setter
+    def autocommit(self, value):
+        """Change this connection autocommit mode.
+
+        :type value: bool
+        :param value: New autocommit mode state.
+        """
+        if value and not self._autocommit:
+            self.commit()
```

Comment:
In accordance with Java implementation - in PEP 249 there is no recommendations for this